fastlane_core/spec/fastlane_pty_spec.rb

```diff
@@ -14,9 +14,15 @@
       end
 
       it 'doesn t return -1 if an exception was raised in the block in PTY.spawn' do
+        status = double("ProcessStatus")
+        allow(status).to receive(:exitstatus) { 0 }
```

Comment:
changing the implementation made this fail and return status code that was -1 on my machine but 1 on others. This is also specific to the command and to the timing of the parsing error vs the execution of the command.

So we make this test a bit less dependent on all these factors to control the way it will be managed by our stack.